### DISCUSSION TITLE
O que esperar de um "Conhecimento avançado em X"

### DISCUSSION THREAD

**Discussion by victorsilent** [frontendbr/forum#1141] [Discussão]
A algum tempo atrás foram criadas duas issues questionando o que faltava nas empresas que a galera mandava CV e o que faltava no o pessoal que mandava CV.
Pois bem, vendo os tópicos acabei não respondendo essa pergunta que me fiz hoje e queria saber de vocês, o que seria um/uma dev com conhecimento avançado em uma tecnologia/prática/ferramenta?
Quando vocês acham que atingiram esse nível, e, para os que acham que não atingiram ainda, o que falta pra você dizer isso de si?

E indo além, para quem contrata, o que esperam de um/uma dev com com esse nivel de conhecimento em algo?

**Comment by rubenmarcus** [comment]
Independencia.  Conseguir tocar um projeto, com a tecnologia exigida sem a ajuda de outro dev, stack overflow, plugins ou o que for. Acho que é o mesmo conceito de senioridade. Claro que um senior consulta muitas vezes o stack, e também blogs, medium ,documentações ,m as acredito que quanto mais experiente, menos frequencia de se consultar..

**Comment by felipefialho** [comment]
@matheusml Aqueles seus textos explicando as coisas cairiam bem aqui 😄

**Comment by matheusml** [comment]
Oi @victorsilent, excelente pergunta.

Se estamos falando de conhecimento **avançado** em uma tecnologia/prática/ferramenta podemos ver por dois ângulos:

### Domínio
O primeiro ângulo é bem na linha do que o @rubenmarcus falou: **independência**. Queremos um desenvolvedor que domine essa ferramenta a ponto de ninguém precisar se preocupar com o rendimento dele e com a certeza de que ele vai entregar tudo **sozinho**.

### Compartilhamento
O segundo ângulo já é um pouco diferente. Além de dominar a ferramenta, o candidato tem que saber também **compartilhar o conhecimento** para que os outros membros da equipe eventualmente cheguem no nível dele.

**Essa pode ser, de forma resumida, a diferença entre um Pleno e um Sênior no seu time.**

Espero ter ajudado 
👍

**Comment by felipefialho** [comment]
E linkando: #982
